models/layerzero/arbitrum/layerzero_arbitrum_send.sql

```diff
@@ -24,30 +25,30 @@ WITH send_detail AS (
         s._adapterParams AS adapter_params,
         s._refundAddress AS refund_address,
         s._zroPaymentAddress AS zro_payment_address,
-        t.from AS user_address,
+        t."from" AS user_address,
         t.to AS transaction_contract_address,
         CAST(t.value AS DOUBLE) AS transaction_value,
-        CASE WHEN len(_destination) >= 82
-            THEN '0x' || right(_destination, 40)
-            ELSE '' END AS local_contract_address, -- 
-        CASE WHEN len(_destination) >= 82
-            THEN substring(_destination, 1, len(_destination) - 40)
+        CASE WHEN bytearray_length(_destination) >= 40
+            THEN bytearray_reverse(bytearray_substring(bytearray_reverse(_destination), 1, 20))
+            ELSE 0x END AS local_contract_address,
+        CASE WHEN bytearray_length(_destination) >= 40
+            THEN bytearray_reverse(bytearray_substring(bytearray_reverse(_destination), 21))
             ELSE _destination END AS remote_contract_address
     FROM {{ source ('layerzero_arbitrum', 'Endpoint_call_send') }} s
     INNER JOIN {{ source('arbitrum','transactions') }} t on t.block_number = s.call_block_number
         AND t.hash = s.call_tx_hash
         {% if not is_incremental() %}
-        AND t.block_time >= '{{transaction_start_date}}'
+        AND t.block_time >= date('{{transaction_start_date}}')
         {% endif %}
         {% if is_incremental() %}
-        AND t.block_time >= date_trunc("day", now() - interval '1 week')
+        AND t.block_time >= date_trunc('day', now() - interval '7' day)
         {% endif %}
     WHERE s.call_success
         {% if not is_incremental() %}
-        AND s.call_block_time >= '{{transaction_start_date}}'
+        AND s.call_block_time >= date('{{transaction_start_date}}')
```

Comment:
minor: the format we've been using is:
`<column> >= TIMESTAMP '{{variable}}'`
which casts the string variable to timestamp type

Reply:
this also applies on all incremental / non incremental blocks in all spells

Reply:
Updated.